src/common-crypto/CryptoBackend.ts

```diff
@@ -195,13 +155,6 @@ export interface OnSyncCompletedData {
     catchingUp?: boolean;
 }
 
-/**
- * Options object for {@link CryptoBackend#checkOwnCrossSigningTrust}.
- */
-export interface CheckOwnCrossSigningTrustOpts {
```

Comment:
We're removing something that wasn't deprecated?

Reply:
Parameters of https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/common-crypto/CryptoBackend.ts#L104 which is deprecated